src/aurelia-i18n.js

```diff
@@ -51,16 +51,17 @@ function configure(frameworkConfig, cb): Promise<void> {
 
   // check whether Intl is available, otherwise load the polyfill
   if (window.Intl === undefined) {
+    console.info('Intl API is not available. Trying to load the polyfill.');  // eslint-disable-line no-console
```

Comment:
We don't want to use console.log here. Instead, we would want to use the logger api. Can you update this PR with that change?